### ISSUE TITLE
How to properly handle variants that can't be annotated?

### ISSUE THREAD

**Issue by thomasyu888** [genome-nexus/genome-nexus#414]
Current Behavior
---
When annotating a MAF file through Genome Nexus we get:

```
	Failed annotations summary:  409 total failed annotations
		Records with HGVSp null variant classification:  0
		Records that failed due to other unknown reason: 409
```

These unknown reasons could include "variants that have reference mismatches" or other issues.  When looking at the annotated MAF file, I see that these 409 "failed" variants have actually been added back.   This is confusing for multiple reasons:

1. The number of columns returned from Genome Nexus can be far greater than the minimal set of MAF columns required for Genome Nexus to work (so you will have columns that have smaller number of columns)
2. If you pass in a fully formed maf file, it is close to impossible to tell which of these variants were unannotated vs annotated

Proposed Feature
---
It would be nice to be able to tell the difference between the variants that annotated properly vs not.

1.  Add an annotation to the annotated maf file (e.g. UNANNOTATED) so that users can look into these variants
2. Only write out the annotatable variants and write out a separate "variants_errored" file.

Personally, I like solution 1.  If we take the example above (variants that have reference mismatches), this is useful to know. One thing to add is that I have an automated process set up to run genome nexus across millions of variants.  Sifting through the logs to see which variants could not be annotated is not a viable solution.

**Comment by n1zea144** [comment]
@thomasyu888 Is it possible to use an existing field in the MAF, maybe "Validation Status"?  Not sure if centers are providing this value (when a MAF is supplied - MSK may).  We can add an agreed upon value to this field to indicate that the annotation failed.

**Comment by thomasyu888** [comment]
@n1zea144 Absolutely, we could definitely use an existing field in the MAF.  I am fine with whichever field that the team thinks makes the most sense to use.
